gamemode/config/config.lua

```diff
@@ -298,16 +298,16 @@ GM.Config.F1MenuHelpPageTitle = "Wiki page"
 -- Put Steam ID's and ranks in this list, and the players will have that rank when they join.
 GM.Config.DefaultPlayerGroups = {
     ["STEAM_0:0:00000000"] = "superadmin",
-    ["STEAM_0:0:11111111"] = "admin",
+    ["STEAM_0:0:11111111"] = "admin"
```

Comment:
Commas after the last entry are done on purpose: people are less likely to make a mistake when they add to the lists.